module/VuFind/src/VuFind/ILS/Driver/Demo.php

```diff
@@ -904,7 +907,8 @@ public function getMyProfile($patron)
             'phone'           => '1900 CALL ME',
             'mobile_phone'    => '1234567890',
             'group'           => 'Library Staff',
-            'expiration_date' => 'Someday'
+            'expiration_date' => 'Someday',
+            'birthdate'       => $birthDate->format('Y-m-d')
```

Comment:
I don't think birthdate is currently part of the driver spec. Would it make sense to add this as a separate PR?

Reply:
Yes, and that was my intention. I expanded the TODO item.